config/config-sil-dc.yaml

```
@@ -57,6 +60,12 @@ active_modules:
       main:
         selftest_success: true
     module: IMDSimulator
+  ovm:
+    config_implementation:
+      main:
+        simulate_error: false
+        simulate_error_delay: 5
+    module: OVMSimulator
```

The `module:` label is topmost with all other modules. Perhaps keep it this way for readability.
```suggestion
  ovm:
    module: OVMSimulator
    config_implementation:
      main:
        simulate_error: false
        simulate_error_delay: 5
```